addons/visibility/stringtable.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ZEN">
+    <Package name="Visibility">
+        <Key ID="STR_ZEN_Visibility_Enabled_Description">
+            <English>Indicates when a player can see the Zeus' cursor</English>
```

Comment:
Needs a period at the end, for consistency with other settings.

Maybe "cursor position" instead of just "cursor", but i am not sure on that.